README.md

```diff
@@ -1,7 +1,7 @@
 Hunter – Hunts Performance Regressions
 ======================================
 
-_This is an unsupported open source project created by DataStax employees._
+_This is open source project was created by DataStax employees._
```

Comment:
nit: `_This open source project was created by DataStax employees._`

Reply:
updated

Reply:
I would remove this entire line in the same PR that we will need to do to change the license headers in every file. Apparently this needs to be done by a Datastax employee. (As Datastax is the donor for the code.)